test/lib/cql_test_env.cc

```diff
@@ -75,6 +75,8 @@ cql_test_config::cql_test_config(shared_ptr<db::config> cfg)
     // which all get thrown away when the test is done. This can cause timeouts
     // if /tmp is not tmpfs.
     db_config->commitlog_use_o_dsync.set(false);
+
+    db_config->add_cdc_extension();
```

Comment:
Doesn't this line add the CDC extension to all tests, not just those that use cdc (as the commit message suggests)?
If that's the decision then I'm fine with it, but then we can avoid leaking this `add_cdc_extension` function from db/config.hh and just do all the magic here, so the interface stays clear.

Reply:
It does and it should. CDC is now enabled by default and this extension is added at the beginning of main.cc. This makes all the tests more similar to production.

Reply:
How are we going to do the logic here? I thought about it but config API does not allow changing the extensions.

Reply:
I was thinking about overriding the default constructor above, which calls make_shared without parameters. It could instead just take the cdc extension and construct a proper config. But it's not a big issue so I'm also fine with leaving the code as is. I'll enqueue later today (afk for a couple of minutes)